crates/polars-plan/src/client/check.rs

```diff
@@ -1,30 +1,15 @@
 use polars_core::error::{polars_err, PolarsResult};
 use polars_io::path_utils::is_cloud_url;
 
-use crate::dsl::Expr;
 use crate::plans::options::SinkType;
-use crate::plans::{DslFunction, DslPlan, FileScan, FunctionIR};
+use crate::plans::{DslPlan, FileScan};
 
 /// Assert that the given [`DslPlan`] is eligible to be executed on Polars Cloud.
 pub(super) fn assert_cloud_eligible(dsl: &DslPlan) -> PolarsResult<()> {
-    let mut expr_stack = vec![];
     for plan_node in dsl.into_iter() {
         match plan_node {
-            DslPlan::MapFunction { function, .. } => match function {
-                DslFunction::FunctionIR(FunctionIR::Opaque { .. }) => {
```

Comment:
This one should still be disallowed. In fact, whole `FunctionIR` is disallowed as we don't implement serde on it.

Reply:
I know - but we don't have to check it explicitly. We will try to serialize it and raise a ComputeError. There are tests for this.

Reply:
Fair enough. :+1: